### PR TITLE
Fixed completion and ":"

### PR DESCRIPTION
**Pull request by cableman**

### CODE REVIEW

File: scripts/itkdev-docker-compose-completion.bash [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #/usr/bin/env bash
 _idc_completions()
 {
+  local cur
+  _get_comp_words_by_ref -n : cur
```
Comment (rimi-itk) [inline scripts/itkdev-docker-compose-completion.bash]: When trying this, I get the error `_get_comp_words_by_ref: command not found`. Are there any requirements that must be fulfilled before using this command?

File: scripts/itkdev-docker-compose-completion.bash [hunk 2]
```diff
@@ -9,6 +12,8 @@ _idc_completions()
   local suggestions=($(compgen -W "url open drush mailhog:open mailhog:url sql:connect sql:port xdebug host:insert images:pull composer" -- "${COMP_WORDS[1]}"))
 
   COMPREPLY=("${suggestions[@]}")
+
+  __ltrim_colon_completions "$cur"
```
Comment (rimi-itk) [inline scripts/itkdev-docker-compose-completion.bash]: When trying this, I get the error `__ltrim_colon_completions: command not found`. Are there any requirements that must be fulfilled before using this command?

File: scripts/itkdev-docker-compose-completion.bash (hunk 1 above)
Comment (cableman) [reply scripts/itkdev-docker-compose-completion.bash]: It's an new part of bash-completion. Found this online:

> Installing bash-completion (1.2) through brew (brew install bash-completion) solved the issue for me, it might work for you too.

File: scripts/itkdev-docker-compose-completion.bash (hunk 2 above)
Comment (cableman) [reply scripts/itkdev-docker-compose-completion.bash]: It's an new part of bash-completion. Found this online:

> Installing bash-completion (1.2) through brew (brew install bash-completion) solved the issue for me, it might work for you too.

File: scripts/itkdev-docker-compose-completion.bash (hunk 1 above)
Comment (cableman) [reply scripts/itkdev-docker-compose-completion.bash]: If that fixes the issue we should add that as an notice to the readme file.

File: php5.6-fpm/Dockerfile [hunk 3]
```diff
@@ -43,6 +43,7 @@ RUN DEBIAN_FRONTEND=noninteractive \
     mysql-client \
     unzip \
     git \
+    imagemagick \
```
Comment (rimi-itk) [inline php5.6-fpm/Dockerfile]: @cableman, why is this added to the PR called “Fixed completion and ":"”?

Comment (cableman) [reply php5.6-fpm/Dockerfile]: Because my git-voodoo failed here
